guide/coaching-company.md

```diff
@@ -135,16 +135,13 @@ could do with your support.
 
 <h2 id="q6">How to become a Coaching Company</h2>
 
-First, register yourself and your coaches here: <a href="http://teams.railsgirlssummerofcode.org">teams.railsgirlssummerofcode.org</a>
+* First, look for a team in your city; this might require reaching out to your local <a href="http://railsgirls.com/">Rails Girls chapter</a> and to development communities such as your local Ruby groups to let them know that you're willing to host a team.
```

Comment:
Rails Girls Chapter is what it's usually referred to within the Rails Girls community + on the site (see eg. here: http://railsgirls.com/organizersweekend). I prefer using that to "orga team" because 1) chapter is independent from the single "events/workshops"
2) the organisational team usually tends to change over the years
3) it's shorter :D